ngraph/test/runtime/interpreter/opset_int_tbl.hpp

```diff
@@ -35,6 +35,7 @@ NGRAPH_OP(LRN, ngraph::op::v0)
 NGRAPH_OP(MVN, ngraph::op::v0)
 NGRAPH_OP(NormalizeL2, op::v0)
 NGRAPH_OP(PriorBox, ngraph::op::v0)
+NGRAPH_OP(Proposal, ngraph::op::v0)
```

Comment:
Two comments about versioning:

- Proposal v0 is matched to Proposal-1 in spec. Is this correct?
- Proposal op class has two versions v0 and v4. This implementation focuses on implementation of v0. v4 has additional output, so that I assume it might need different implementation. Which opset is currently implementing CPU plugin for instance? If the answer is v0, should we also take care of v4 ref impl in the future?
@lazarevevgeny could you comment on this?

Reply:
We should focus on v4 version of the op because CPU supports v4. Also, the v0 version is a sub-set of the v4, so it should be possible to re-use v4 implementation in the v0 (I am not sure about this).

Reply:
Sure thing, would you think it would be an acceptable solution to make the v0 ref impl. call the v4 implementation? That's how I see implementing it with the least code duplication for now, of course I can implement it separately too if that will be better.

Reply:
Sure, I will be also aligning in terms of handling the output termination, to CPU.